python/ray/_private/worker.py

```diff
@@ -3189,7 +3189,7 @@ def remote(__t: type) -> Any:
 @overload
 def remote(
     *,
-    num_returns: Union[int, float] = Undefined,
+    num_returns: Union[int, float, Literal["dynamic"]] = Undefined,
```

Comment:
"dynamic" is replaced by "streaming", could you update here?

Also can you remove `float` since it cannot be float :)

Reply:
should now be addressed